astropy/convolution/tests/test_convolve.py

```diff
@@ -65,6 +65,18 @@ def test_tuple(self):
         assert_array_almost_equal_nulp(z,
             np.array([0., 3.6, 5., 5.6, 5.6, 6.8, 0.]), 10)
 
+    def test_quantity(self):
```

Comment:
Does this need parameterization like the other tests?
```
    @pytest.mark.parametrize(('boundary', 'nan_treatment',
                              'normalize_kernel', 'preserve_nan', 'dtype'),
                             itertools.product(BOUNDARY_OPTIONS,
                                               NANHANDLING_OPTIONS,
                                               NORMALIZE_OPTIONS,
                                               PRESERVE_NAN_OPTIONS,
                                               VALID_DTYPES))
```

Reply:
I used the first couple of functions on test_convolve.py as a template, and so thought parameterizing it would be unnecessary - but this could be added instead of the equality check.

Reply:
@keflavich , what do you think?

Reply:
I'd default to parametrizing the same way.  In principle, there's no interaction between what's being tested and the other parameters, but that's sort of the point of the test - make sure that's true and we never regress.